tests/tests.py

```diff
@@ -11,7 +11,7 @@
 class DNSTest(unittest.TestCase):
 
     def setUp(self):
-        self.channel = pycares.Channel(timeout=1.0, tries=1)
+        self.channel = pycares.Channel(timeout=5.0, tries=1)
```

Comment:
Sometimes the tests on Travis CI failed (inconsistently) on the domains I used as examples for fragmented TXT records. I believe a timeout of 1 second was okay for google.com (used in the tests previously), but not necessarily for "less provisioned domains" that might not always respond within that timespan.